### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by sergio-usma**
### Project requirements
- [X]  A new project was established with webpack based on the completed webpack exercise.
- [X]  An index.html file was created, and HTML markup was written.
- [X]  A To-Do List placeholder _(ul element)_ was created.
- [X] The index.html file was set as a template using the HTML Webpack Plugin.
- [X]  An index.js file was created, and an array of simple to-do tasks (array of objects) was set.
- [X]  Each task object contained three keys: description [string], completed [bool], and index [number].
- [X]  A function was written to iterate over the tasks array and populate an HTML list item element for each task.
- [X]  On page load, the dynamically created list of tasks was rendered in the dedicated placeholder.
- [X]  The list appeared in order of the index values for each task.
- [X]  A style.css file was created, and rules for the To-Do List were set.
- [X]  CSS was loaded by Webpack Style/CSS Loader.
- [X]  The list was a clone of the part of the minimalist project captured in the video below.
- [X]  All source files (index.html, index.js, and style.css) were in the /src directory.
- [X]  Distribution files were generated by webpack and served by the webpack dev server from the /dist folder.

### CODE REVIEW

File: src/modules/renderList.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    this.toDoList.push(task);
+    this.renderList();
+    this.toLocalStorage();
+  }
+
+  renderList() {
+    const listContainer = document.querySelector('.list__container__checklist');
+    listContainer.innerHTML = '';
+
+    // eslint-disable-next-line no-restricted-syntax
```
Comment (emmiiorji) [inline src/modules/renderList.js]: Kindly avoid disabling linter checks in line. They represent coding standards that you share with your team. You can use a forEach() method here.

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @sergio-usma, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<div align='center'>

</div>

## Good Highlights✔️

- Webpack is used to set up the project
- Rendering tasks from array works as expected


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/renderList.js (hunk 1 above)
Comment (sergio-usma) [reply src/modules/renderList.js]: - [X] forEach method replaced!

Comment (mikethreels) [inline src/modules/renderList.js]: - [x] Like the previous CR mentioned please avoid disabling linters, as they are used to make your code better and easier to read. if something is not clear or if you have a question please leave a comment in the PR.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Sergio, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- good job setting up webpack

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/renderList.js (hunk 1 above)
Comment (sergio-usma) [reply src/modules/renderList.js]: Oh, man, I forgot to delete the comment. So, I will solve this right now.

**Review by osamaashraf6 — APPROVED** [review]
Hi @sergio-usma 😊, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
MinionsGIF (3)

## Highlights🎉🎉

- [ ] Good setting up the project using webpack💯
- [ ] Use webpack css loader🚀🟠
- [ ] Descriptive pull request👍


## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**